unique.c

```diff
@@ -0,0 +1,34 @@
+/* Amanda Cuevas - Identifies whether a string has all unique characters, or whether there is only one of each character in the string. */
```

Comment:
Works perfectly, but again - do not come to class expecting me or Ms. Hsu to fix your non-working code.